client/state/sharing/selectors.js

```diff
@@ -51,6 +51,7 @@ export function getAvailableExternalAccounts( state, service ) {
 					keyringConnection.additional_external_users.map( externalUser => ( {
 						ID: externalUser.external_ID,
 						name: externalUser.external_name,
+						description: externalUser.external_description,
 						picture: externalUser.external_profile_picture,
```

Comment:
We don't have any data for the picture yet, this is why it's a white box for now